apps/web/src/features/ui/emoji-picker/_index.scss

```
@@ -1,11 +1,17 @@
 @import "src/styles/vars_mixins";
 
 .emoji-picker-dialog {
-  position: absolute;
   z-index: 201;
 
   em-emoji-picker {
     width: 300px;
+    // Set by the `size` middleware from the room actually left in the viewport,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)